templates/_asset.html

```diff
@@ -1,40 +1,34 @@
 <div class="col-4 p-card">
   {% if asset.data.image %}
-  <a href="/{{ asset.file_path }}" target="_blank">
-    <img
-      class="p-card__thumbnail p-image--bordered asset-thumbnail"
-      src="/{{ asset.file_path }}"
-    />
-  </a>
-  <hr class="u-sv1" />
+    <a href="https://assets.ubuntu.com/v1/{{ asset.file_path }}"
+       target="_blank">
+      <img class="p-card__thumbnail p-image--bordered asset-thumbnail"
+           src="https://assets.ubuntu.com/v1/{{ asset.file_path }}"/>
+    </a>
+    <hr class="u-sv1" />
   {% endif %}
   <div class="p-card__content">
-    <p><strong>{{ asset.file_path }}</strong></p>
+    <p>
+      <strong>{{ asset.file_path }}</strong>
+    </p>
     <p>
       <span>
-      <a href="/manager/update?file-path={{ asset.file_path }}">Tags</a>:
+        <a href="/manager/update?file-path={{ asset.file_path }}">Tags</a>:
       {% for tag in asset.tags %}
         <a href="/manager?tag={{ tag.name }}">
-          <span class="p-label">{{tag.name}}</span>
+          <span class="p-label">{{ tag.name }}</span>
         </a>
       {% endfor %}
-      </span>
-    </p>
-    <p><span>Created: {{ asset.created }}</span></p>
-    <p>
-      <small>Copy:</small>
-      <button
-        class="p-button--positive copy-link is-inline"
-        data-url="/{{ asset.file_path }}"
-      >
-        Link
-      </button>
-      <button
-        class="p-button js-copy-image-tag"
-        data-url="/{{ asset.file_path }}"
-      >
-        Image tag
-      </button>
-    </p>
-  </div>
+    </span>
+  </p>
+  <p>
+    <span>Created: {{ asset.created }}</span>
+  </p>
+  <p>
+    <small>Copy:</small>
+    <button class="p-button--positive copy-link is-inline"
+            data-url="/{{ asset.file_path }}">Link</button>
```

Comment:
Does this need the prefix?

Reply:
yes it's temporary as the new project isn't linked to the current Swift server (files server), once the project is deployed, I will remove this prefix, adding it to make sure that we have imported the assets successfully.

Reply:
Isn't Ant saying maybe you need to add the link prefix here as well as on line 3?

Reply:
why can't we just link it to the swift server for testing? or let it fail gracefully if the asset is missing?

Reply:
> Isn't Ant saying maybe you need to add the link prefix here as well as on line 3?

Oh sorry about that, I miss read Ant's question.

> why can't we just link it to the swift server for testing? or let it fail gracefully if the asset is missing?

I added the prefix temporarily where it will be removed as soon this is deployed. The reason for this was that at that time I wasn't sure whether we will keep the current swift server or not.
This is outdated, I will update this PR and remove the prefix as we can link it to the swift server for testing.

Reply:
:+1: thanks